roles/tendrl-ansible.tendrl-server/tasks/tendrl-monitoring-integration.yml

```diff
@@ -32,6 +32,13 @@
     owner: apache
     group: apache
 
+- name: Allow apache to log messages in graphite-web
```

Comment:
Is this required due to redirection we are using?

Reply:
graphite itself is using some apache redirections, so we need to give permission for apache to log messages in /var/log/grphite-web, graphite-web integration depending on apache call only.

Reply:
Was this a problem with previous graphite build?

Reply:
@mbukatov @TimothyAsirJeyasing @mbukatov the problem which I saw inside /var/log/graphite_web there is two log files are present info.log and error.log, so these files are created by python-django at the time of initialization command. So it actually creating files with root permission. So httpd unable to log the messages into it. so we need to change this file owner to apache.

Here is the question is why this has not happened with old graphite?
   Actually, the old graphite initialization command is not creating any log files under /var/log/graphite_web, so httpd is created log files without any issue.

So this change is required.

Reply:
it is same this what we are doing for /var/lib/graphite-web/graphite.db, because we are running command as root user

Reply:
@GowthamShanmugam would it make sense to just run the graphite db init as apache user then?